tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm

```diff
@@ -327,6 +339,48 @@ sub content {
   return $html;
 }
 
+sub prettify_phenotypes {
+  my ($self, $entries, $species) = @_;
+  my @result;
+
+  my $hub = $self->hub;
+
+  #unify
+  my %phenotypes;
+  foreach my $entry (@$entries) {
+    $entry =~ tr/_/ /;
+    my @parts = split('\+',$entry);
+    $phenotypes{$parts[0]}{$parts[2]}{$parts[1]} = 1;
+  }
+
+  # display fromat:   # 'HYPERTENSION__ESSENTIAL+MIM_morbid+ENSG00000135744' -> HYPERTENSION ESSENTIAL(ENSG00000135744,MIM_morbid) OR  HYPERTENSION ESSENTIAL(ENSG00000135744,MIM_morbid & Orphanet & DDG2P)
+  foreach my $pheno (keys %phenotypes){
+    foreach my $object (keys %{$phenotypes{$pheno}}){
```

Comment:
Wondering if it would be useful to sort the `(keys %phenotypes)` and/or the `(keys %{$phenotypes{$pheno}})`

Reply:
Thanks for checking @ens-lgil. The final input will be sorted by the _display_items_list_ method 
 when the list gets collapsed: https://github.com/Ensembl/ensembl-webcode/blob/release/95/modules/EnsEMBL/Web/Component/Shared.pm#L1726-L1751
For the uncollapsed list (1-3 phenotypes) this will not be sorted indeed, but hopefully not too much of a problem.